corehq/apps/reports/standard/cases/basic.py

```diff
@@ -111,45 +130,75 @@ def get_special_owner_ids(self, admin, unknown, demo, commtrack):
     @property
     @memoized
     def case_owners(self):
+        """
```

Comment:
@esoergel @sheelio Added this documentation to bring to notice what case owners does and would do. Mostly is from the existing implementation but just want to flag couple of things.

> for selected reporting group ids, returns user_ids belonging to these groups
       also finds the sharing groups which has any user from the above reporting group

The second half does not sound right since that would involve all submission by a sharing group which the user just belongs to but has not submitted the cases, hence returning more cases

> selected user ids
       also finds the sharing groups which has any user from the above reporting group

Same as above.

> ids and descendants ids of selected locations and assigned locations of users selected

Similar issue with finding locations of selected users returning a larger set of cases.